hosted/aks/helper/helper_cluster.go

```diff
@@ -564,6 +580,26 @@ func DeleteAKSClusteronAzure(clusterName string) error {
 	return nil
 }
 
+// ShowAKSStatusOnAzure returns the status of AKS based on the passed query
+// For e.g. '.currentKubernetesVersion' will only return the current kubernetes version of the cluster
+func ShowAKSStatusOnAzure(clusterName, resourceGroup, query string) (out string, err error) {
+	fmt.Println("Showing AKS cluster ...")
+	args := []string{"az", "aks", "show", "--subscription", subscriptionID, "--name", clusterName, "--resource-group", resourceGroup, "|", "jq", "-r", query}
+	cmd := strings.Join(args, " ")
+	fmt.Printf("Running command: %s\n", args)
+	out, err = proc.RunW("bash", "-c", cmd)
```

Comment:
shouldn't we use the same approach as before? like:
```suggestion
	out, err = proc.RunW("az", args...)
```
(args/cmd would need to be adapted as well)

Reply:
We are using `jq -r` to parse JSON with a specific query, so I had used `bash`, but I noticed `az` also has a `--query` parameter that serves the same purpose.  I will use that for both places. Thanks for the review.

Reply:
Ah, right, I did not notice the jq stuff but indeed it should be doable over `--query`